config/k8s-cron-template.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: scheduled-job
+spec:
+  template:
+    metadata:
+      name: scheduled-job
+    spec:
+      containers:
+      - name: index.docker.io/sheltertechsf/askdarcel-api:latest
```

Comment:
I have experience with Kubernetes but not Google Kubernetes Engine; is there a way to ensure that the image tag is consistent between the api pods and this job? My fear here is that the way this is written, any time someone merges to master, we'll build a new image and push it to `latest`, and in production we'll start running whatever the latest image is, not the latest *deployed* image.

I'm more familiar with using [Helm](https://helm.sh/) to parameterize your Kubernetes YAML files and ensure that the image tags are the same across multiple pods that are part of the same "release". For example, taking this open source Grafana Helm Chart, you can specify the image in one place in [values.yaml](https://github.com/helm/charts/blob/bb5b5ab85caf47966d02ed24ec64f6a751b46956/stable/grafana/values.yaml#L290) while referring to it in multiple locations [[1]](https://github.com/helm/charts/blob/bb5b5ab85caf47966d02ed24ec64f6a751b46956/stable/grafana/templates/deployment.yaml#L92) [[2]](https://github.com/helm/charts/blob/bb5b5ab85caf47966d02ed24ec64f6a751b46956/stable/grafana/templates/deployment.yaml#L111).